crates/driver/src/infra/config/cli.rs

```diff
@@ -34,6 +36,42 @@ pub struct Args {
     /// The time to allocate to generating quotes, in milliseconds.
     #[clap(long, env, default_value = "5000")]
     pub quote_timeout_ms: u64,
+
+    /// The mempools to use when submitting settlements.
+    #[clap(
+        long,
+        env,
+        default_value = "public",
+        value_enum,
+        use_value_delimiter = true
+    )]
+    pub mempools: Vec<Mempool>,
+
+    /// The Flashbots API URL.
+    #[clap(
+        long,
+        env,
+        use_value_delimiter = true,
+        default_value = "https://rpc.flashbots.net"
+    )]
+    pub flashbots_api_urls: Vec<Url>,
+
+    /// The solver address used to sign transactions. Expects a 20-byte hex
+    /// encoded string. This can't be specified along with --solver-private-key,
+    /// exactly one of them must be specified.
```

Comment:
`clap` supports specifying this [with argument groups](https://docs.rs/clap/latest/clap/_derive/_tutorial/index.html#argument-relations).